apps/mull-ui/src/app/app.tsx

```diff
@@ -41,6 +43,16 @@ export const App = () => {
             <EventPage event={dummyEvent} prevPage={'Review'} />
           </div>
         </Route>
+        <Route path={ROUTES.HOME}>
+          <div className="page-container">
+            <SubNavigationBar />
+            <SwipeableRoutes>
+              <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
+              <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
+              <Route path={ROUTES.MY_EVENTS} component={() => <div>MY_EVENTS!</div>} />
```

Comment:
I got one last suggestion here. This is to avoid people routing to /home/somethingelse
```suggestion
              <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
              <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
              <Route path={ROUTES.MY_EVENTS} component={() => <div>MY_EVENTS!</div>} />
              <Route path={ROUTES.HOME} render={() => <Redirect to={ROUTES.DISCOVER} />} />
```